contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2791,14 +2801,19 @@ rewriteBatchLevelStatement(
 		auto cctx = ctx->create_or_alter_trigger()->create_or_alter_ddl_trigger();
```

Comment:
Please add testcase for ddl trigger

Reply:
We aren't supporting ddl triggers. But can I add in the test scenarios:

Reply:
Let's do the followings:
1. add a comment here explaining `create_or_alter_ddl_trigger` is not supported for now.
2. add a test case so that a future developer can recognize this corner case.